orion/sites/admin/src/achieve/achieve.styl

```diff
@@ -11,3 +13,43 @@
     margin: spacing4x 0
     line-height: lineHeight3
     font-size: fontSize3
+
+  & ul
+    list-style: none
+    margin: 0
+    padding: 0
+
+
+#view-achieve #list-heading
+  border-bottom: 2px solid colorGrayDark
+
+  & li
+    display: inline-block
+    height: spacing4x
+    line-height: spacing4x
+
+
+#view-achieve #achieve-row
+  border-bottom: 1px solid colorGrayDark
+
+  & li
+    display: inline-block
+    height: spacing8x
+    line-height: spacing8x
+
+
+#view-achieve
+  & .li-small
+    width: 80px
```

Comment:
I think I did this too in `programs.styl`. Could we change this to `spacing10x` instead of `80px`? You will find the variable in `statics.styl`